lib/constants.py

```diff
@@ -162,27 +162,27 @@ def repo_path(*args):
 
 AUTH_TYPE = {
     "development": "t2.micro",
-    "production": "m5.xlarge",
+    "production": "m5.2xlarge",
     "ha-development": "t2.micro",
 }
 
 ########################
 # Machine Cluster Sizes
 AUTH_CLUSTER_SIZE = { # Auth Server Cluster is a fixed size
-    "development" : 1,
+    "development": 1,
     "production": 1, # should be an odd number
     "ha-development": 1,  # should be an odd number
 }
 
 CONSUL_CLUSTER_SIZE = { # Consul Cluster is a fixed size
-    "development" : 1,
+    "development": 1,
     "production": 5, # can tolerate 2 failures
     "ha-development": 3,  # can tolerate 1 failures
 }
 
 VAULT_CLUSTER_SIZE = { # Vault Cluster is a fixed size
-    "development" : 1,
-    "production": 3, # should be an odd number
+    "development": 1,
+    "production": 2,
```

Comment:
Why is this no longer odd? Doesn't it have to be odd so a master can be elected?

Reply:
> What is the spdb submodule update pulling in?

It's pulling in hvac pinned at a specific version.  There is also a note that says we need to remove this pin when we merge in refactor.

Reply:
https://www.vaultproject.io/docs/internals/high-availability.html consul needs an odd number.  Vault doesn't.